templates/angular/igx-ts/projects/empty/index.ts

```diff
@@ -10,6 +10,21 @@ class EmptyJsBlocksProject implements ProjectTemplate {
 	public projectType: string = "igx-ts";
 	public hasExtraConfiguration: boolean = false;
 
+	private CUSTOM_THEME
+		= `/* See: https://www.infragistics.com/products/ignite-ui-angular/angular/components/themes.html */
+@import "~igniteui-angular/core/styles/themes/index";
+
+$primary: #731963 !default;
+$secondary: #ce5712 !default;
+
+$app-palette: igx-palette($primary, $secondary);
+
+@include igx-core();
+@include igx-theme($app-palette);
+	`;
+	private DEFAULT_THEME = `,
+			  "node_modules/igniteui-angular/styles/igniteui-angular.css"`;
```

Comment:
The tabs in this string will also mess the formatting.

Reply:
Done